CONTRIBUTING.md

```diff
@@ -281,29 +281,29 @@ To manually create a backport, follow [the standard pull request process][pr-cre
 ## How to review pull requests
 [pr-review]: #how-to-review-pull-requests
 
-The Nixpkgs project receives a fairly high number of contributions via GitHub pull requests.
-Reviewing and approving these is an important task and a way to contribute to the project.
+The Nixpkgs project receives a high number of pull requests.
+Reviewing and approving these is an important way to contribute to the project.
 
-The high change rate of Nixpkgs makes any pull request that remains open for too long subject to conflicts that will require extra work from the submitter or the merger.
-Reviewing pull requests in a timely manner and being responsive to the comments is the key to avoid this issue.
-GitHub provides sort filters that can be used to see the [most recently](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) and the [least recently](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc) updated pull requests.
-We highly encourage looking at [this list of ready to merge, unreviewed pull requests](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+review%3Anone+status%3Asuccess+-label%3A%222.status%3A+work-in-progress%22+no%3Aproject+no%3Aassignee+no%3Amilestone).
+The high change rate makes any PR that remains open for too long subject to merge conflicts.
+To avoid extra work, reviewing PRs timely and being responsive is key.
+GitHub provides sort filters to see the [most recently updated](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) pull requests.
+We highly encourage looking at [this list of ready to merge, unreviewed pull requests](https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr%20is%3Aopen%20review%3Anone%20status%3Asuccess%20no%3Aproject%20no%3Aassignee%20no%3Amilestone%20).
```

Comment:
This changed url redirects back

Reply:
It is also missing the `-label:"2.status: work-in-progress"` in the search field, making any changes the search loose that search term

Reply:
What's the motivation for replacing `+` with `%20`?

Also, why a trailing `%20`?